plugins/action/ci_script.py

```diff
@@ -188,12 +195,16 @@ def run(self, tmp=None, task_vars=None):
             {"_raw_params": script_path_str, "chdir": output_dir.as_posix()}
         )
 
-        return self._shared_loader_obj.action_loader.get(
-            "ansible.builtin.script",
-            task=file_task,
-            connection=self._connection,
-            play_context=self._play_context,
-            loader=self._loader,
-            templar=self._templar,
-            shared_loader_obj=self._shared_loader_obj,
-        ).run(task_vars=task_vars)
+        if not dry_run:
```

Comment:
And if-else with a couple of return statements is asking for simplification using a single if as a barrier for the simpler one:
```
if dry_run:
  return {"command": json.dumps(ansible_encoding.decode_ansible_raw(task_args))}

return self._shared_loader_obj.action_loader.get(
    "ansible.builtin.script",
    task=file_task,
    connection=self._connection,
    play_context=self._play_context,
    loader=self._loader,
    templar=self._templar,
    shared_loader_obj=self._shared_loader_obj,
).run(task_vars=task_vars)
```

Reply:
sure